### DISCUSSION TITLE
Use CLI with Pycharm Terminal and GitHub action to run it to test a set of GraphQL APIs.

### DISCUSSION THREAD

**Discussion by rajarajan1985** [thunderclient/thunder-client-support#1097] [General]
Can I able to use CLI alone in pycharm (terminal) without the actual extension? Also, I would like to run this CLI to test graphql endpoints. Is there any sample or documentation available to follow?

**Comment by rangav** [comment]
Yes you can use CLI separately without extension. But Node should be installed on your system 

Documentation 
https://rangav.medium.com/d91eb5c71d8e

**Reply by rajarajan1985** [reply]
Gotcha. Thanks for the quick response. Let me follow the documentation and will reach out if  I have any questions.

**Reply by rangav** [reply]
You are Welcome, Sure let me know

**Reply by rajarajan1985** [reply]
How does the below command recognize the collection name? Where must this collection/requests file be placed in the project folder? What is the file format? 
npx tc --col 'collectionName'

**Reply by rangav** [reply]
You need to use extension to create collections 

then you can use extension or cli to add requests to collections 

then you can run collections from CLI

**Reply by rajarajan1985** [reply]
ah okay. As I mentioned earlier, I'm using pycharm, not VS code. That's why I asked whether I could use the CLI alone without extension.

**Reply by rangav** [reply]
Oh ok you cannot use all the features without extension

**Reply by rajarajan1985** [reply]
hmm got it. Thanks for your help. :)
